lms/djangoapps/hera/admin.py

```diff
@@ -13,7 +13,32 @@ class OnboardingAdmin(admin.ModelAdmin):
     """
     Reflection of the hera model on the admin panel.
     """
-    pass
+
+    def get_queryset(self, request):
+        """
+        Create a default onboarding object in case we have none yet.
+        """
+        qs = super(OnboardingAdmin, self).get_queryset(request)
+
+        if not qs:
+            default_onboarding = self.model()
+            default_onboarding.save()
+
+            qs = super(OnboardingAdmin, self).get_queryset(request)
+
+        return qs
+
+    def has_add_permission(self, request):
+        """
+        Allow to add Onboarding only if no objects exist.
+        """
+        return not self.model.objects.exists()
+
+    def get_actions(self, request):
+        """
+        Remove 'delete_selected' action from the list of available actions.
+        """
+        return {}
```

Comment:
Is it really correct?

Reply:
@drtwisted @sendr

Reply:
Yes, there was an OrderedDict, I was wrong with my comment above.

Reply:
Should we change the docsting?

Reply:
@drtwisted please do it.

Reply:
done.